timeout.go

```diff
@@ -0,0 +1,83 @@
+package handel
+
+import (
+	"sync"
+	"time"
+)
+
+// TimeoutStrategy decides when to start a level in Handel. It is started and
+// stopped by the Handel structure. A basic strategy starts level according to a
+// linear timeout function thanks to the Handel.StartLevel method. More advanced
+// strategies could for example implement the Pre/PostProcessor interface,
+// register itself as a processor to Handel, and start a level according to
+// specific rules such as "all nodes answered with a 1-contribution
+// multi-signature", etc.
+type TimeoutStrategy interface {
+	Start()
```

Comment:
If I understand correctly it allows to have a timeout depending on how long the previous level took, which is good.